### ISSUE TITLE
Drop support for Ruby 1.8.7

### ISSUE THREAD

**Issue by codener** [makandra/geordi#79]
Since Ruby 1.8 is EOL by far now, let's drop support. In the process, we should simplify the code and remove any 1.8-targeting parts.

**Comment by denzelem** [comment]
We might need to freeze `thor` in the gemspec for a last release compatible with Ruby < 2.0.0 before.

Last compatible `thor` version: `0.20.3` (last release before `1.0.0` where Ruby < 2.0.0 support ist dropped)

Otherwise you will get an error like this:

```
$ ruby -v
ruby 1.9.3p551 (2014-11-13 revision 48407) [x86_64-linux]
$ gem install geordi -v '2.10.1'
ERROR:  Error installing geordi:
	thor requires Ruby version >= 2.0.0.
```

Workaround:

```
gem install thor -v '0.20.3'
gem install geordi
```

**Comment by codener** [comment]
Thanks for pointing this out. The Ruby-2+-Geordi will be released as 3.0.0, so it will be simple to add more 2.x releases at any time.

**Comment by denzelem** [comment]
Done with https://github.com/makandra/geordi/tree/v3.0.0.
